RN 1.2.0 (#165)
percona/everest-doc: docs/release-notes/Percona-Everest-1.2.0-(2024-10-01).md

- [EVEREST-1372](https://perconadev.atlassian.net/browse/EVEREST-1372): We’ve resolved the issue that prevented us from deleting the monitoring endpoint that is not currently in use.

- [EVEREST-1427](https://perconadev.atlassian.net/browse/EVEREST-1427) - 
PostgreSQL no longer gets stuck in an unknown state despite having duplicate backup storage in different namespaces.

- [EVEREST-1287](https://perconadev.atlassian.net/browse/EVEREST-1287) - Before Percona Everest 1.2.0, every database node had to be scheduled on a separate K8s node within an EKS distribution. Otherwise, the database would fail to start if it didn't have a separate K8s node available. Now, it is preferable, but the databases will still come up even if no separate K8s node is available, and this setting is applied regardless of the distribution being used.

## Known limitations

- RBAC will not work if you have configured Single sign-on (SSO) and your identity provider (IdP) is **Microsoft Entra**.

- If you remove permissions for a resource, some permissions will still be valid even after they have been removed. The new set of permissions will only take effect when you refresh the page.

> - When you upgrade to Percona Everest 1.2.0, the database transitions to the initializing state as part of the PostgreSQL operator upgrade process. However, this initializing state does not cause any downtime.

Reviewer: ```suggestion
- When you upgrade PostgreSQL operators to version 2.4.1, the database transitions to the initializing state as part of the upgrade process. However, this initializing state does not cause any downtime.
```

> - When you upgrade PXC operators to version 1.15.0, single node PXC databases will be restarted, resulting in downtime. However, its worth noting that single node databases are not typically used in production environments.

Reviewer: Maybe we should mention MySQL instead
```suggestion
- When you upgrade PXC operators to version 1.15.0, single node MySQL databases will be restarted, resulting in downtime. However, its worth noting that single node databases are not typically used in production environments.
```

> - When you upgrade PXC operators to version 1.15.0, single node PXC databases will be restarted, resulting in downtime. However, its worth noting that single node databases are not typically used in production environments.

Reviewer: Let's be assertive and let users know that they shouldn't use these DBs in prod
```suggestion
- When you upgrade PXC operators to version 1.15.0, single node MySQL databases will be restarted, resulting in downtime. However, its worth noting that single node databases should not be used in production environments.
```

> - When you upgrade PXC operators to v1.15.0, if your PXC databases consist of a single node, they will be restarted, resulting in downtime. However, its worth noting that single node databases are not typically used in production environments.

Reviewer: ```suggestion
- When you upgrade PXC operators to version 1.15.0, single node PXC databases will be restarted, resulting in downtime. However, its worth noting that single node databases are not typically used in production environments.
```
